src/main/java/mods/eln/Eln.java

```diff
@@ -253,13 +253,30 @@
     public int electricalInterSystemOverSampling;
 
     public CopperCableDescriptor copperCableDescriptor;
+    public GraphiteDescriptor GraphiteDescriptor;
 
+    /*public ElectricalCableDescriptor creativeCableDescriptor;
+    public ElectricalCableDescriptor T2TransmissionCableDescriptor;
+    public ElectricalCableDescriptor T1TransmissionCableDescriptor;*/
     public ElectricalCableDescriptor veryHighVoltageCableDescriptor;
```

Comment:
Change the 'normal' qualities to start with 'normalQuality'.

Also, do not start variable names with capital letters. That's for constants only.